spec/reek/configuration/app_configuration_spec.rb

```diff
@@ -46,20 +46,33 @@
     end
   end
 
+  describe '.exclude_paths' do
+    let(:config_path) { 'spec/samples/configuration/with_excluded_paths.reek' }
+
+    it 'should return all paths to exclude' do
+      with_test_config(config_path) do
+        expect(described_class.exclude_paths).to eq [
+          'spec/samples/source_with_exclude_paths/ignore_me',
+          'spec/samples/source_with_exclude_paths/nested/ignore_me_as_well'
+        ]
+      end
+    end
+  end
+
   describe '.load_from_file' do
     it 'loads the configuration from given file' do
-      Reek::Configuration::AppConfiguration.load_from_file(sample_configuration_path)
-      expect(Reek::Configuration::AppConfiguration.configuration).to eq(sample_configuration_loaded)
+      described_class.load_from_file(sample_configuration_path)
+      expect(described_class.configuration).to eq(sample_configuration_loaded)
     end
   end
 
   describe '.reset' do
     it 'resets the configuration' do
-      Reek::Configuration::AppConfiguration.load_from_file(sample_configuration_path)
+      described_class.load_from_file(sample_configuration_path)
 
-      expect(Reek::Configuration::AppConfiguration.configuration).to eq(sample_configuration_loaded)
-      Reek::Configuration::AppConfiguration.reset
-      expect(Reek::Configuration::AppConfiguration.configuration).to eq(default_configuration)
+      expect(described_class.configuration).to eq(sample_configuration_loaded)
+      described_class.reset
+      expect(described_class.configuration).to eq(default_configuration)
```

Comment:
Not really part of this PR but cleaned up according to the good old boyscout rule.:)